docs/source/gbd2017_models/risk_attributable_causes/chronic_kidney_disease/index.rst

```diff
@@ -134,11 +134,6 @@ Initialization:
 Progress:
 -As simulants age, their risk exposure will change, which may result in them progressing into a different disease state over time. Also, simulants will experience mortality based on their risk exposure.
 
-.. todo::
-Confirm with the RT and SE team how this model affects other models.
-
-- This model affects other models – disability weights, excess mortality weight for each state, cause specific mortality rate. 
-
```

Comment:
Since the treatment for LDL-C does not impact IKF, the relevant disabilities for this project will not be affected.  However, disability due to CKD will be affected by the methodology described above.  If simulants progress into subsequently worse CKD states too slowly, there may be a reduction of disability for the population (caveat: it is possible that slowing the progression into worse states may lessen the disability if said progression results in faster death).  Punchline: if morbidity is lower than GBD, we should expect mortality to be higher.
If mortality from CKD is elevated due to these assumptions, we should see a correlated reduction in IHD and IS deaths since the simulants will die from CKD instead of the CV causes.